packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -1174,29 +1197,41 @@ class Auth {
 
     /**
      * A wrapper method for commerce-sdk-isomorphic helper: authorizeIDP.
+     * Initiates OAuth2 authorization flow for Identity Provider (IDP) login.
      *
      */
     async authorizeIDP(parameters: AuthorizeIDPParams) {
-        const redirectURI = parameters.redirectURI || this.redirectURI
+        const slasClient = this.client
         const usid = this.get('usid')
-        const customParameters = extractCustomParameters(parameters)
-        const {url, codeVerifier} = await helpers.authorizeIDP(
-            this.client,
-            {
-                redirectURI,
-                hint: parameters.hint,
-                ...(usid && {usid}),
-                ...customParameters
-            },
-            this.isPrivate
```

Comment:
Do we use this anywhere else? Is this safe to remove?

Reply:
Oh, that's a good catch actually. Let's put it back in. I'm not the most familiar with IDP login setup but looking at the `helpers.authorizeIDP` method, it takes in an optional `privateClient` property that determines whether the underlying authorization request includes a code challenge or not (we include one for public client and omit for a private client)

https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/blob/main/src/static/helpers/slasHelper.ts#L222